test/plugin/scenarios/jdk-threadpool-scenario/bin/startup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+home="$(cd "$(dirname $0)"; pwd)"
+
+java -jar ${agent_opts} ${home}/../libs/jdk-threadpool-scenario.jar &
```

Comment:
Your new test is not added into GHA.

Reply:
> GHA

I don't ever develop a plugin. How to add it into GHA? Can you teach me? Thanks!

Reply:
`plugin-tests-*.yaml` in https://github.com/apache/skywalking-java/tree/main/.github/workflows. Last step in the doc, https://github.com/apache/skywalking-java/blob/main/docs/en/setup/service-agent/java-agent/Plugin-test.md#local-test-and-pull-request-to-the-upstream

Reply:
> `plugin-tests-*.yaml` in https://github.com/apache/skywalking-java/tree/main/.github/workflows. Last step in the doc, https://github.com/apache/skywalking-java/blob/main/docs/en/setup/service-agent/java-agent/Plugin-test.md#local-test-and-pull-request-to-the-upstream

Ok, I will see it. Thank you!